app/components/sbom/app-logo/index.scss

```diff
@@ -0,0 +1,14 @@
+.app-logo-container {
+  width: 40px;
+  height: 40px;
+  padding: 0.5em;
+  border-radius: var(--sbom-app-logo-border-radius);
+  border: 1px solid var(--sbom-app-logo-border-color);
+  box-sizing: border-box;
+
+  img {
+    object-fit: contain;
+    background-image: none;
```

Comment:
What is the function of this css declaration?

Reply:
It contains image within the container